source/TimePicker.js

```diff
@@ -470,10 +492,11 @@
 					if (this.meridiemEnable === true) {
 						this.createComponent(
 							{classes: 'moon-date-picker-wrap', components:[
-								{kind:'moon.MeridiemPicker', name:'meridiem', classes:'moon-date-picker-field', value: valueHours > 12 ? 1 : 0, meridiems: this.meridiems || ['am','pm'], onChange: 'meridiemPickerChanged'},
+								{kind:'moon.MeridiemPicker', name:'meridiem', classes:'moon-date-picker-field', meridiems: this.meridiems || [{name: 'AM', start: '00:00', end: '11:59'}, {name: 'PM', start: '12:00', end: '23:59'}], max: this.meridiems.length - 1 || 1, onChange: 'meridiemPickerChanged'},
 								{name: 'meridiemLabel', content: this.meridiemText, classes: 'moon-date-picker-label moon-divider-text'}
 							]}
 						);
+						this.$.meridiem.value = this.$.meridiem.getMeridiemIndex(values.hour, values.minute);
```

Comment:
We should use `getMeridiemIndex()` to find meridiem value.
However, this method is API of moon.MeridiemPicker.
So I placed this line after creation.

And to ignore notice, I assigned its value instead of using setter.